ppr-ui/src/views/mhrInformation/MhrInformation.vue

```diff
@@ -203,35 +205,35 @@ export default defineComponent({
     const parseCurrentOwnerGroups = async (): Promise<void> => {
       const { data } = await fetchMhRegistration(getMhrInformation.value.mhrNumber)
       const currentOwnerGroups = data?.ownerGroups || [] // Safety check. Should always have ownerGroups
+
+      // Store a snapshot of the existing OwnerGroups for baseline of current state
+      await setMhrTransferCurrentHomeOwnerGroups(cloneDeep(data.ownerGroups))
+
       // Create an ID to each individual owner for UI Tracking
       // TODO: Remove after API updates to include the ID for Owners
```

Comment:
This TODO comment is not needed anymore.

Reply:
Will remove

Reply:
Removed